assets/blocks/course-outline/status-store.js

```diff
@@ -0,0 +1,226 @@
+import { select as selectData, registerStore } from '@wordpress/data';
+import { createReducerFromActionMap } from '../../shared/data/store-helpers';
+import { Status } from './status-control';
+import { select, controls } from '@wordpress/data-controls';
+
+const DEFAULT_STATE = {
+	completedLessons: new Set(),
```

Comment:
Considering it's for the store, I think we could use an [array here](https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state). WDYT?

Reply:
Oh. Didn't knew this limitation of Redux. Too bad because a Set was fitting here. 75434b9eeb3f99f5e22ecc130a540357827bcab8